php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -259,4 +259,22 @@ public static boolean isNullOrEmptyString(ExpressionTree tree) {
     return false;
   }
 
+  public static Tree getParentOfKind(Tree tree, Kind kind) {
+    return getParentOfKind(tree, kind, null);
+  }
+
+  public static Tree getParentOfKind(Tree tree, Kind kind, @Nullable Kind[] terminators) {
```

Comment:
You may also define this method with [varargs](https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html) and drop `getParentOfKind(Tree tree, Kind kind)`:
 ```
public static Tree getParentOfKind(Tree tree, Kind kind, Kind... terminators) {
```